packages/terra-status-view/tests/nightwatch/status-view-spec.js

```diff
@@ -0,0 +1,99 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { resizeTo } = require('terra-toolkit/lib/nightwatch/responsive-helpers');
+
+module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
```

Comment:
convert to webdriver.io tests.  Use badge as an example these should be pretty straightforward a11y and screenshot tests.